config/tools-manual.json

```diff
@@ -69,5 +69,9 @@
     "Others": {
         "description": "The following is a list of tools that comes under Other category",
         "toolsList": []
+    },
+    "CLIs": {
```

Comment:
followed approach that all categories, even empty are here in manual file so people that add manually have easier life